glass/core/array.py

```diff
@@ -1,26 +1,27 @@
 # author: Nicolas Tessore <n.tessore@ucl.ac.uk>
 # license: MIT
-"""module for array utilities"""
+"""Module for array utilities."""
 
-import numpy as np
 from functools import partial
 
+import numpy as np
+
 
 def broadcast_first(*arrays):
     """Broadcast arrays, treating the first axis as common."""
     arrays = tuple(np.moveaxis(a, 0, -1) if np.ndim(a) else a for a in arrays)
     arrays = np.broadcast_arrays(*arrays)
-    arrays = tuple(np.moveaxis(a, -1, 0) if np.ndim(a) else a for a in arrays)
-    return arrays
+    return tuple(np.moveaxis(a, -1, 0) if np.ndim(a) else a for a in arrays)
 
 
 def broadcast_leading_axes(*args):
-    """Broadcast all but the last N axes.
+    """
+    Broadcast all but the last N axes.
 
     Returns the shape of the broadcast dimensions, and all input arrays
     with leading axes matching that shape.
 
-    Example
+    Example:
```

Comment:
This should probably "Examples" as in the [numpydoc format](https://numpydoc.readthedocs.io/en/latest/format.html#examples), but no `:` at the end.